src/main/python/utils/dave_engine.py

```diff
@@ -2051,7 +2048,8 @@ def nan_and_inf_to_num (obj):
         if np.isnan(obj):
             return 0
 
-    elif isinstance(obj, np.ndarray):
+    elif isinstance(obj, np.ndarray) \
+        and len(obj.shape) == 1:
```

Comment:
What happens where the object has a more complicated shape? Is it safe to ignore it?

Reply:
This was a quick fix for the workshop. But probably we need to add support to other shapes. I think that the only method that returns shapes biggers than 1 is the phaseogram.